ideas/README.md

```diff
@@ -0,0 +1,141 @@
+# Ideas
+
+## Intro
+
+Welcome to the Google Summer of Code ideas repository!
+Here you can propose new ideas to participate to this fantastic event 
+and contribute to the nix/nixOS/nixpkgs community ecosystem!
+
+Below you will find all instructions to propose your ideas, so don't be shy
+to contribute!
+
+Summerily yours,
+the repo maintainers.
+
+## Submission fields explanation
+
+### ID
+The ID is composed by the current year and an incremental 4 digit integer
+assigned by one of the repo maintainers and will also be the directory name
+that will contain all the artifacts for that idea.
+
+### Title
+The title should be clear, concise and inspirative.
+
+It should also be prefixed by a categorey chosen among:
+    * nix
+    * nixpkgs
+    * nixos
+Depending on which area of the ecosystem more closely fits your idea.
+
+### Author
+Just the author's github handle.
+
+### Proposed Menthor
+If you're in contact with a member of the nix community who's interested in
+helping you as your direct menthor, you can mention the github handle here so
+that the repo maintainers wil be added to the conversation in the PR.
+
+### Effort
+This is the "size" of the effort level of the idea. It should be one of:
+    * SMALL
+    * MEDIUM
+    * LARGE
+
+### Difficulty
+This is how "hard" it is at a technical level to implement the idea. It can be:
+    * EASY
+    * MEDIUM
+    * HARD
+### Overview/Abstract
+This is a short description of the idea. Try to stay within the limit of 400
+characters. Think of this as your "elevator pitch".
+### Description
+Here you can add all the nitty gritty details of the idea. Feel free to add
+anything that you think is relevant for the staff to understand and evaluate
+your idea.
+### Linkography
+Just the link to the relevant documentation and/or description you wrote
+above.
+
+## Idea Template
+
+This would be a PR aimed at creating a README.md file with the content below
+in the directory [ideas/2024/0000](./ideas/2024/0000).
```

Comment:
Putting every idea in it's own directory will make it hard to read through all of them at once because you will have to open different pages for each one.
It should be a ideas-*list* so I think we should just put them in one file and sort them by size.
Please remember that this should be a low level entry point with ideas for newbies.
It should also be easy and minimal effort for mentors to propose ideas, we have the [RFC](https://github.com/nixos/rfcs/) process for complex and large proposals.

Reply:
yes but the RFC process is not related t other GSoC initiative.

I agree about the idea list and that's why I wanted to keep the README.md as a "summary"/"index" which then points to the different directories that can contain all the needed data, including additional files, images, source code files, etc..

Maybe I'm over-optimizing :)